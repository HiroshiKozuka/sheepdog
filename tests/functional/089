#!/bin/bash

# Test a combination of object reclaim and discard

. ./common

for i in 0 1 2; do
    _start_sheep $i
done
_wait_for_sheep 3
_cluster_format -c 2:1

$DOG vdi create test 16M
dd if=/dev/zero | $DOG vdi write test
$DOG vdi snapshot test

$DOG vdi delete test -s 1
_node_info

$QEMU_IO -c "discard 0 16m" sheepdog:test | _filter_qemu_io
_node_info
