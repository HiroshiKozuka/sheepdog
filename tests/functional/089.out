QA output created by 089
using backend plain store
STORE	DATA	VDI	VMSTATE	ATTR	LEDGER	STALE
0	4	2	0	0	4	0
1	4	2	0	0	4	0
2	4	2	0	0	4	0
discard 16777216/16777216 bytes at offset 0
16 MiB, X ops; XX:XX:XX.X (XXX YYY/sec and XXX ops/sec)
STORE	DATA	VDI	VMSTATE	ATTR	LEDGER	STALE
0	0	2	0	0	0	0
1	0	2	0	0	0	0
2	0	2	0	0	0	0
